### PR TITLE
Project Completion 

### PR DESCRIPTION
**Pull request by AsimKhan2019**
Completed the budget application project (Snack Time), adhering to the technical requirements. Achieved the following tasks: 

Able to add new users using Devise 
Able to sign in users with Devise 
Able to add new categories
Able to add payments against each category
Able to view the complete payments of each category 
Able to view the sums of each category 
Able to view the total sum of all categories 
Able to retrieve the relevant categories of each user
Able to display the relevant payments of each user with their respective categories
Created RSpec tests
Applied navigations to the application.
Added a splash screen after login on the home page

 The user can successfully use the application and navigate through the application to add categories and their relevant payments. Using the design provided by Microverse and limited screen options and designs, much was unclear about the real design without a working sample.

I realize that there is so much to learn from this massively awesome framework, and I will put in a great deal of time and effort to learn and explore more about Ruby on Rails. 

Kindly please approve my pull request. 

My presentation video is below. 

https://drive.google.com/file/d/1DOHOYa9eahs2vXW9TCor6EijzeWsvoe-/view?usp=share_link

My Live App

https://ror-budget-capstone.herokuapp.com/users/sign_in

### CODE REVIEW

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @AsimKhan2019, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice video presentation 👏 👏
- The App is working fine 👏 👏
- Linter is green (the last commit has a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow **was first used correctly**  (Pull Request is made from the feature branch into the dev branch) 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,24 +1,76 @@
...
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Microverse LazyCoders Coding Partners Inspiration etc
+
+📝 License
+This project is MIT licensed.
```
[the remark this answers is not in the document]
Comment (AsimKhan2019) [reply README.md:76]: @OLIPLICHE I will connect the MIT to the license link.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,76 @@
...
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Microverse LazyCoders Coding Partners Inspiration etc
+
```
[the remark this answers is not in the document]
Comment (AsimKhan2019) [reply README.md:74]: @OLIPLICHE I will add acknowledgments in this section

File: README.md [hunk 3]
```diff
@@ -1,24 +1,76 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
[the remark this answers is not in the document]
Comment (AsimKhan2019) [reply README.md:2]: @OLIPLICHE I will move the link into the Read Me File

File: app/views/shared/_payment.html.erb [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<div class="transaction-widget-cont">
+  <div class="transaction-widget">
+    <%unless @externals%>
+      <div class="group-image thumb">
```
[the remark this answers is not in the document]
Comment (AsimKhan2019) [reply app/views/shared/_payment.html.erb:4]: @OLIPLICHE Thank you

File: app/views/payments/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<div class="cat-bar">
+  <div class="content">
```
[the remark this answers is not in the document]
Comment (AsimKhan2019) [reply app/views/payments/index.html.erb:2]: @OLIPLICHE Thank you

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (AsimKhan2019) [reply README.md:76]: @OLIPLICHE I have merged my MIT License with the dev branch

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (AsimKhan2019) [reply README.md]: @OLIPLICHE I have added acknowledgements to the Read Me File

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (AsimKhan2019) [reply README.md:2]: @OLIPLICHE I have added link to the Read Me File

**Review by DammyShittu — APPROVED** [review]
Hi @AsimKhan2019,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AsimKhan2019** [comment]
@DammyShittu Thank you for the approval

**Pull request merged**